### PR TITLE
Added hyperlink to Hot Exit for more details

### PR DESCRIPTION
**Pull request by khubaibalam2000**
This Pull Request closes the issue mentioned in #193312 by attaching hyperlink of Vscode documentation to the hot exit description, hence adding more details to the description.

### CODE REVIEW

File: src/vs/workbench/contrib/files/browser/files.contribution.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ const hotExitConfiguration: IConfigurationPropertySchema = isNative ?
...
 			nls.localize('hotExit.onExitAndWindowClose', 'Hot exit will be triggered when the last window is closed on Windows/Linux or when the `workbench.action.quit` command is triggered (command palette, keybinding, menu), and also for any window with a folder opened regardless of whether it\'s the last window. All windows without folders opened will be restored upon next launch. A list of previously opened windows with unsaved files can be accessed via `File > Open Recent > More...`')
 		],
-		'description': nls.localize('hotExit', "Controls whether unsaved files are remembered between sessions, allowing the save prompt when exiting the editor to be skipped.", HotExitConfiguration.ON_EXIT, HotExitConfiguration.ON_EXIT_AND_WINDOW_CLOSE)
+		'description': nls.localize('hotExit', "[Hot Exit](https://code.visualstudio.com/docs/editor/codebasics#_hot-exit) controls whether unsaved files are remembered between sessions, allowing the save prompt when exiting the editor to be skipped.", HotExitConfiguration.ON_EXIT, HotExitConfiguration.ON_EXIT_AND_WINDOW_CLOSE)
```
Comment (bpasero) [inline src/vs/workbench/contrib/files/browser/files.contribution.ts:133]: This needs to be `markdownDescription`

File: src/vs/workbench/contrib/files/browser/files.contribution.ts [hunk 2]
```diff
@@ -140,7 +140,7 @@ const hotExitConfiguration: IConfigurationPropertySchema = isNative ?
 			nls.localize('hotExit.off', 'Disable hot exit. A prompt will show when attempting to close a window with editors that have unsaved changes.'),
 			nls.localize('hotExit.onExitAndWindowCloseBrowser', 'Hot exit will be triggered when the browser quits or the window or tab is closed.')
 		],
-		'description': nls.localize('hotExit', "Controls whether unsaved files are remembered between sessions, allowing the save prompt when exiting the editor to be skipped.", HotExitConfiguration.ON_EXIT, HotExitConfiguration.ON_EXIT_AND_WINDOW_CLOSE)
+		'description': nls.localize('hotExit', "[Hot Exit](https://code.visualstudio.com/docs/editor/codebasics#_hot-exit) controls whether unsaved files are remembered between sessions, allowing the save prompt when exiting the editor to be skipped.", HotExitConfiguration.ON_EXIT, HotExitConfiguration.ON_EXIT_AND_WINDOW_CLOSE)
```
Comment (bpasero) [inline src/vs/workbench/contrib/files/browser/files.contribution.ts:143]: This needs to be `markdownDescription`

**Review by bpasero — CHANGES_REQUESTED** [review]
Unfortunately we cannot just use direct links to our website, it needs to be a forward link, but I can create one and fill it in later.

**Comment by khubaibalam2000** [comment]
@bpasero Yes, I figured out that URL link should be something like https://aka.ms/vscode-hot-exit, but I don't have the permission to do this. Is there anything I can do to make this commit resolvable?

**Comment by bpasero** [comment]
@khubaibalam2000 here you go: https://aka.ms/vscode-hot-exit

**Comment by khubaibalam2000** [comment]
@bpasero Thank you for the link. I have added this link as forward URL in the hot exit description in the latest commit.

**Review by bpasero — APPROVED** [review]

**Review by dbaeumer — APPROVED** [review]

**Pull request merged**